op_builder/npu/inference.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+from enum import IntEnum
+import torch
+from .builder import NPUOpBuilder
+
+try:
+    import torch_npu
+except ImportError as e:
+    pass
```

Comment:
@CurryRice233 importing torch here causes problems for DeepSpeed install. Can you please move the `torch` and `torch_npu` imports inside of a class method in this file? (See https://github.com/microsoft/DeepSpeed/blob/9500ab7d476aaf60ec2c76653c67bf5feb758f9e/op_builder/transformer_inference.py#L20 for an example)

The reason for this is that `op_builder` is imported by `setup.py` (before any requirements are installed) and if `torch` is not available the install will fail. Please let me know if you have any questions

Reply:
Moving the `import torch' to the interior of the `try' solves this problem. If I move inside of class method, I need to import torch into each method, which I don't think is a good idea.